src/main/flight/imu.c

```diff
@@ -109,6 +109,7 @@ quaternion offset = QUATERNION_INITIALIZE;
 
 // absolute angle inclination in multiple of 0.1 degree    180 deg = 1800
 attitudeEulerAngles_t attitude = EULER_INITIALIZE;
+float imuAttitudeQuaternion[3]; // quaternion for blackbox 0 - x, 1 - y, 2 - z. For normalized quaternion w = 1
```

Comment:
Actually, normalized quaternion is unit vector: $w = sqrt(1 - (x^2 + y^2 + z^2))$